### DISCUSSION TITLE
OS

### DISCUSSION THREAD

**Discussion by bukiru** [eduardogsilva/wireguard_webadmin#12] [General]
and how can i run it under BSD system?

**Comment by eduardogsilva** [comment]
I'm not quite sure of how complete is the docker support on bsd. If you manage to get it working without major issues, I believe that the container would just run fine. 
I found this https://wiki.freebsd.org/Docker 

If you allow a suggestion, just try docker on a linux VM, maybe it will be easier and more reliable than trying to install docker on bsd.


Let me know if you have any further questions! 
(I will convert this issue to a discussion)

**Comment by eduardogsilva** [comment]
You could try to run the django install without docker, but I believe that some OS calls could fail. 

The firewall would not work since its heavily based on iptables. Maybe some other OS calls would not work too. 
You can try, but I don't think that it worth all the trouble. 
I'm also updating this project frequently, so if you modify to work with bsd, any update would be a pain to manage.

Here is a quick tip of how to run this project without docker: https://github.com/eduardogsilva/wireguard_webadmin/discussions/2
